dataland-frontend/src/components/general/SelectReportingPeriodDialog.vue

```diff
@@ -43,12 +53,31 @@ export default defineComponent({
       if (this.mapOfReportingPeriodToActiveDataset) {
         const sortedReportingPeriodMetaInfoPairs = Array.from(
           (this.mapOfReportingPeriodToActiveDataset as Map<string, DataMetaInformation>).entries(),
-        ).sort((firstEl, secondEl) => compareReportingPeriods(firstEl[0], secondEl[0]));
+        ).sort((firstElement, secondElement) => compareReportingPeriods(firstElement[0], secondElement[0]));
         for (const [key, value] of sortedReportingPeriodMetaInfoPairs) {
+          const answeredDataRequestId = this.answeredDataRequests
+            ?.filter((answeredDataRequest: StoredDataRequest) => {
+              return answeredDataRequest.reportingPeriod == key;
```

Comment:
if there can possibly be multiple answered dataRequestId's then the name of the variable should reflect the plural "IDs", otherwise the filter function is not needed, since you could just access the dataset with the matching reporting period.
I guess the first option is true, correct?

Reply:
Yes, there can be multiple answeredDataRequestIds. Changed it.